crates/e2e-tests/src/tests.rs

```diff
@@ -54,11 +55,25 @@ async fn test_wallet_api() -> Result<(), Box<dyn std::error::Error>> {
         "59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d"
     ))?;
 
-    let capabilities: BTreeMap<U256, BTreeMap<String, BTreeMap<String, Vec<Address>>>> =
-        provider.client().request_noparams("wallet_getCapabilities").await?;
-
     let chain_id = U256::from(provider.get_chain_id().await?);
 
+    let capabilities = {
+        let mut innermost_tree = BTreeMap::new();
+        innermost_tree.insert(
+            "addresses".to_string(),
+            vec![Address::from_str(&std::env::var("DELEGATION_ADDRESS").unwrap()).unwrap()],
+        );
+
+        let mut inner_tree = BTreeMap::new();
+        inner_tree.insert("delegation".to_string(), innermost_tree);
+
+        let mut capabilities = BTreeMap::new();
+        capabilities.insert(chain_id, inner_tree);
+
+        capabilities
+    };
```

Comment:
we should remove the capabilities structs entirely, we don't need them anymore, and instead use a const address here or an env var

Reply:
Just to clarify, you mean to remove `WalletCapabilities` right?

```
#[derive(Debug, Clone, Eq, PartialEq, Deserialize, Serialize)]
pub struct WalletCapabilities(pub HashMap<U64, Capabilities>);
```


I see its not really being used anywhere else in the code

Reply:
yes